kolibri/plugins/coach/assets/src/views/recent-page/report-list.vue

```diff
@@ -54,8 +54,8 @@
       'content-icon': require('kolibri.coreVue.components.contentIcon'),
       'progress-bar': require('kolibri.coreVue.components.progressBar'),
     },
-    computed: {
-      reportList() {
+    watch: {
```

Comment:
`.sort` doesn't return a new array, it sorts in place. The case being that nothing is being done to the array other than sorting it, and this piece of state is only used on this page, I thought this would be alright. Thoughts?